pennylane/gradients/parameter_shift_hessian.py

```diff
@@ -740,17 +744,19 @@ def _param_shift_hessian_tuple(
             instead of evaluating the input tape, reducing the number of device invocations.
 
     Returns:
-        tensor_like or tuple[tensor_like] or tuple[list[QuantumTape], function]:
-
-        - If the input is a QNode with a single trainable argument, a tensor representing the
-          Hessian of size ``(*QNode output dimensions, *QNode input dimensions, *QNode input dimensions)``
-          is returned.
+        function or tuple[list[QuantumTape], function]:
 
-        - If the input is a QNode with multiple trainable arguments, a tuple of Hessian tensors is
+        - # TODO: If the input is a QNode, an object representing the Hessian (function) of
```

Comment:
what is the TODO here?

Reply:
Ah, right! I am not entirely sure the statement is true :sweat_smile: 
I also found a bug while trying to find out, which I have to put up still :)

Reply:
Fixed this :)